lib/events.js

```diff
@@ -141,20 +116,13 @@ EventEmitter.prototype.emit = function emit(type, ...args) {
       err.context = er;
       throw err;
     }
-    return false;
```

Comment:
Why did this get removed?

Reply:
It has moved here https://github.com/nodejs/node/pull/17403/files/426afc70832c47f18f9afd9f90f35c1e554c8d3a#diff-0b93f5d72b0d34f5b972b661935344c3R415

The only case when this code was reached was when a domain was present, other situation throw.